edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -252,11 +252,30 @@ iframe.rocket-chat-view {
 
 .login-providers {
   display: flex;
-  text-align: left;
-  background: $inverse;
+  text-align: center;
+  gap: 2em;
   padding: 10px 0;
   justify-content: center;
   align-items: center;
+  flex-basis: 30%;
+
+  & > .button-container {
+    flex-basis: 50%;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: space-between;
+    background-color: #EDF6F9;
```

Comment:
What do you think set this color as a variable?

Yo can see an example in `/edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_variables.scss`